sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/RecognizePiiEntitiesAsync.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Configuration;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Sample demonstrates how to recognize the Personally Identifiable Information entities of document.
+ */
+public class RecognizePiiEntitiesAsync {
+    /**
+     * Main method to invoke this demo about how to recognize the Personally Identifiable Information entities of document.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        TextAnalyticsAsyncClient client = new TextAnalyticsClientBuilder()
+            .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_API_KEY")))
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TEXT_ANALYTICS_ENDPOINT"))
+            .buildAsyncClient();
+
+        // The document that needs be analyzed.
+        String document = "Microsoft employee with ssn 859-98-0987 is using our awesome API's.";
+
+        client.recognizePiiEntities(document).subscribe(
+            entityCollection -> entityCollection.forEach(entity -> System.out.printf(
+                "Recognized personal identifiable information entity: %s, entity category: %s, entity sub-category: %s, score: %f.%n",
```

Comment:
Some places say "Recognized Personally Identifiable Information entity" I prefer this one too, but whichever you pick, better to have consistency in the samples